unit_test/test_crypt/test_crypt.c

```diff
@@ -150,7 +153,7 @@ int main(void)
 {
     int return_value = 0;
 
-    if (libspdm_cryptest_main() != 0) {
+    if (libspdm_cryptest_main() == false) {
```

Comment:
Don't compare with bool (true/false).

just use `!libspdm_cryptest_main()`.

Reply:
Thanks. I have changed it.